docs/notes/vineyard-operator.rst

```diff
@@ -157,8 +157,8 @@ the vineyard daemon server:
    * - image
      - string
      - The image name of vineyardd image.
-     - "vineyardcloudnative/
-       vineyardd:alpine-latest"
+     - | "vineyardcloudnative/
+       | vineyardd:alpine-latest"
```

Comment:
Looks quite strange. I guess the key won't be very long? Seems a bit over-splitted.